scss/_regions/_footer.scss

```diff
@@ -102,7 +102,8 @@
       }
 
       .social-icon {
-        margin: ($base-spacing / 2);
+        margin: 8px;
+        padding: 1px 1px 0;
```

Comment:
😱! This screams "magic number" to me. Could you add a comment explaining the hack and maybe an issue to remind us to revisit this again later?